avro-serializer/src/main/java/io/confluent/kafka/serializers/AbstractKafkaAvroSerDe.java

```diff
@@ -35,6 +34,7 @@
   protected static final String SCHEMA_REGISTRY_URL = "schema.registry.url";
   protected static final String MAX_SCHEMAS_PER_SUBJECT = "max.schemas.per.subject";
   protected static final String SPECIFIC_AVRO_READER = "specific.avro.reader";
+  protected static final String GENERIC_AVRO_DATUM_WRITER_CLASS = "generic.avro.writer.class";
```

Comment:
@ewencp Yep, saw that PR, would be beneficial for this. Actually have a local branch that uses `AbstractConfig` and `getConfiguredInstance` instead so I will happily update for #162 when it's merged.